cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -212,6 +214,7 @@ func NewCommand() *cobra.Command {
 	command.Flags().BoolVar(&allowOutOfBoundsSymlinks, "allow-oob-symlinks", env.ParseBoolFromEnv("ARGOCD_REPO_SERVER_ALLOW_OUT_OF_BOUNDS_SYMLINKS", false), "Allow out-of-bounds symlinks in repositories (not recommended)")
 	command.Flags().StringVar(&streamedManifestMaxTarSize, "streamed-manifest-max-tar-size", env.StringFromEnv("ARGOCD_REPO_SERVER_STREAMED_MANIFEST_MAX_TAR_SIZE", "100M"), "Maximum size of streamed manifest archives")
 	command.Flags().StringVar(&streamedManifestMaxExtractedSize, "streamed-manifest-max-extracted-size", env.StringFromEnv("ARGOCD_REPO_SERVER_STREAMED_MANIFEST_MAX_EXTRACTED_SIZE", "1G"), "Maximum size of streamed manifest archives when extracted")
+	command.Flags().BoolVar(&managedByArgo, "managed-by-argo", env.ParseBoolFromEnv("ARGOCD_REPO_SERVER_MANAGED_BY_ARGO", false), "Set managed-by label for every k8s resource generated by argo")
```

Comment:
If you want `Current Status: Alpha (Since v2.8.0)` you'd have to add it here and run `make codegen`, so that the docs matches what is in the command.

You could also attempt to add the exact same string in `docs/operator-manual/server-commands/argocd-repo-server.md` without running `make codegen`, but if they differ you'll bump into the same issue as previously.